Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14.2 AS build-fe
+WORKDIR /src/
+COPY ui  .
```

Comment:
tip, if you want future builds to be faster: copy first `ui/package.json` and `ui/yarn.lock`, then run `yarn`, then copy the full directory, then run `yarn build`. That way the `node_modules` folder will stay in the Docker layer cache (as long as the dependencies don't change)